ui/elyria/src/services/c2dcrm.interface.ts

```diff
@@ -10,6 +10,11 @@ export interface CrmListCustomersResponse {
   id: string;
   pda_url: string;
   status: 'new' | 'claimed';
```

Comment:
Add 1 more. Status is `new`, `claimed`, `unverified`

Elyria warned that they may have unverified records.